tests/console/consoletest_setup.pm

```diff
@@ -60,8 +60,9 @@ sub run() {
     select_console 'root-console';
 
     type_string "chown $username /dev/$serialdev\n";
-    # Export the existing status of running tasks for future reference (fail would export it again)
+    # Export the existing status of running tasks and system load for future reference (fail would export it again)
     script_run "ps axf > /tmp/psaxf.log";
+    script_run "cat /proc/loadavg > /tmp/loadavg_consoletest_setup";
```

Comment:
it was more like lack of fantasy to come up with a better name, e.g. could have been "loadavg_before_any_potential_later_fail_to_which_we_want_to_compare"